synapse/models/person.py

```diff
@@ -460,6 +476,25 @@ def getModelDefs(self):
                         'doc': 'The web account from which the contact list was extracted.',
                     }),
                 )),
+
+                ('ps:contact:type:taxonomy', {}, ()),
+                ('ps:skill', {}, (
+
+                    ('name', ('str', {'lower': True, 'onespace': True}), {
+                        'doc': 'The name of the skill.'}),
+
+                    ('type', ('ps:skill:type:taxonomy', {}), {
+                        'doc': 'The type of skill as a taxonomy.'})
+                )),
+
+                ('ps:proficiency', {}, (
```

Comment:
This kinda feels like a light edge currently

Reply:
This will eventually contain secondary properties such as a potential rating value or time that they gained proficiency.